todoman/model.py

```diff
@@ -567,6 +568,7 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             with a ``-`` prepended will be used to sort in reverse order.
         :param bool reverse: Reverse the order of the todos after sorting.
         :param int due: Return only todos due within ``due`` hours.
+        :param start: Return only todos before/after ``start`` date
```

Comment:
This doesn't make it very clear what this arg exactly is. I really do think we need separate `start_before` and `start_after` args.

Opinion, @untitaker ?